.circleci/config.yml

```diff
@@ -3,7 +3,10 @@ version: 2
 jobs:
   build:
     docker:
-      - image: googleapis/nox:0.18.2
+      - image: gcr.io/cloud-devrel-kokoro-resources/python:latest
+        auth:
+            username: _json_key
+            password: $GCR_CREDENTIALS
```

Comment:
@busunkim96 Hmm, that causes [CircleCI to fail](https://circleci.com/gh/googleapis/google-cloud-python/8641):

```
Starting container gcr.io/cloud-devrel-kokoro-resources/python:latest
  image cache not found on this host, downloading gcr.io/cloud-devrel-kokoro-resources/python:latest

Error response from daemon: unauthorized: You don't have the needed permissions to perform this operation, and you may have invalid credentials. To authenticate your request, follow the steps in: https://cloud.google.com/container-registry/docs/advanced-authentication
```

Reply:
It looks like I didn't get the permissions for container registry correct. I'll keep trying. 🤔